### PR TITLE
viewfield2 b

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW/NOT READY
- (Edit the above to reflect status)

# Summary
- TL;DR - what's this PR for?

# Review By (Date)
- When does this need to be merged by?

# Urgency
- How critical is this PR?

# Steps to Test

1. Do this
1. Then this
2. Then this

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: src/Plugin/rest/resource/ReactParagraphsResource.php [hunk 1]
```diff
@@ -129,13 +129,11 @@ public function get($entity_type_id, $bundle) {
...
-          [
-            $form[$field_name],
-            $field_config,
-            $data[$field_name],
-          ]
+        $this->moduleHandler->alter(
+          'react_paragraphs_form_field_data',
+          $data[$field_name],
+          $form[$field_name],
+          $field_config
```
Comment (sherakama) [inline src/Plugin/rest/resource/ReactParagraphsResource.php:136]: This order changed. Will have to swap it in stanford_news.

**Review by sherakama — APPROVED** [review]

File: src/Plugin/rest/resource/ReactParagraphsResource.php (hunk 1 above)
Comment (pookmish) [reply src/Plugin/rest/resource/ReactParagraphsResource.php:136]: yeah. it seemed more inline with other alter hooks. the first argument is the one you wanna alter

Comment (sherakama) [reply src/Plugin/rest/resource/ReactParagraphsResource.php:136]: I'm onboard with that. Thanks for quality control.
